docs/conf.py

```diff
@@ -0,0 +1,72 @@
+project = 'Polyfactory'
+copyright = "2023, Starlite API"
+author = "Starlite API"
+release = '2.0.0'
+
+extensions = [
+    "sphinx.ext.autodoc",
+    'sphinx.ext.napoleon',
+]
+
+templates_path = ['_templates']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+napoleon_google_docstring = True
+napoleon_include_special_with_doc = True
+napoleon_use_admonition_for_examples = True
+napoleon_use_admonition_for_notes = True
+napoleon_use_admonition_for_references = False
+napoleon_attr_annotations = True
+
+suppress_warnings = [
+    "autosectionlabel.*",
+    "ref.python",  # TODO: remove when https://github.com/sphinx-doc/sphinx/issues/4961 is fixed
+]
+
+html_theme = "pydata_sphinx_theme"
+html_static_path = ["_static"]
+html_css_files = ["style.css"]
+html_js_files = ["versioning.js"]
+html_favicon = "images/favicon.ico"
+html_logo = "images/logo.svg"
+html_show_sourcelink = False
+html_sidebars = {"about/*": []}
+html_title = "Starlite Project | Polyfactory"
```

Comment:
```suggestion
html_static_path = ["_static"]
html_css_files = ["style.css"]
html_show_sourcelink = False
html_title = "Starlite Project | Polyfactory"
```

Not needed after switching to our custom Sphinx theme